src/health-check.sh

```diff
@@ -5,10 +5,17 @@ CONFIGTYPE="${CONFIGTYPE:-"fhem.cfg"}"
 TELNETPORT="${TELNETPORT:-7072}"
 STATE=0
 
-if [ "${CONFIGTYPE}" != "configDB" ] && [ -s ${FHEM_DIR}/${CONFIGTYPE} ] && [ -z "$(cat ${FHEM_DIR}/${CONFIGTYPE} | grep -P "^define .* telnet ${TELNETPORT}")" ]; then
+RUNNING_INSTANCES=$(pgrep -f "/bin/sh -c /health-check.sh" | wc -l)
+
+if [ "${RUNNING_INSTANCES}" -gt "1" ]; then
+  echo "Instance already running, aborting another one"
```

Comment:
echoing here does not make sense as it will replace status appearance of the existing instance in docker status overview.

Reply:
From the docs

> To help debug failing probes, any output text (UTF-8 encoded) that the command writes on stdout or stderr will be stored in the health status and can be queried with docker inspect. Such output should be kept short (only the first 4096 bytes are stored currently).

This output should help, if there is already a running instance of the health check to report this, that something isn't correct at all.

Reply:
@jpawlowski 

How we can proceed here?